src/App.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
 import './App.scss';
+// eslint-disable-next-line import/extensions
+import Sum from './components/Sum.jsx';
```

Comment:
remove extension and linter will be ok
```suggestion
import Sum from './components/Sum';
```